### ISSUE TITLE
Optimize Training Robustness

### ISSUE THREAD

**Issue by kamil271e** [iwamaciek/stwp#24]
- [x] Use Train, Val and Test set for: 2020, 2021, 2022 - data processor needs to be modified
- [x] RobustScalers / StandardScalers as a default instead of MinMaxScalers -> set it int BaselineRegressor class
- [x] Use bigger grid sizes limiting the evaluation to the confines of Poland
- [ ] ~~Optimize data preprocessing~~
- [ ] ~~Check whether normalization before fitting improves baselines performance~~
- [x] Test different loss functions - L1 Loss is better than L2, try: HuberLoss, LogCosh
- [x] Test different regularization techniques: Dropout, BatchNorm
